src/test/java/htsjdk/samtools/seekablestream/SeekableBufferedStreamTest.java

```diff
@@ -69,6 +69,20 @@ public void testRandomRead() throws IOException {
         assertEquals(buffer1, buffer2);
     }
 
+    @Test
+    public void testReadExactlyOneByteAtEndOfFile() throws IOException {
+        try (final SeekableStream stream = new SeekableHTTPStream(new URL(BAM_URL_STRING))){
```

Comment:
almost, still missing the space between the `){`

Reply:
whoops, didn't see your comment until after I merged